server_src/runners/history.js

```diff
@@ -346,45 +326,37 @@ class HistoryCollection {
       return null;
     }
   }
+
+  /**
+   * Grabs the timelapse by downloading it from OctoPrint's API
+   * @param fileName
```

Comment:
I like breathing room in files and function annotations. How about you?

Reply:
I agree